Django_web/manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+

```

Comment:
To ignore `.pyc` files and `__pycache__` folder use `.gitignore` file please
For inspiration, you can use https://github.com/github/gitignore/blob/master/Python.gitignore or https://www.toptal.com/developers/gitignore.
And yes, please exclude the `static` folder (except your app static files if you have them). Just I see, that it's only admin and grappeli, don't need to include these static files in your repo. 
And it will easier to review :)

Reply:
`.gitignore` added and 'staric` deleted. 
Yes, that's better, thanks)